common/performance.py

```
@@ -0,0 +1,721 @@
+        Returns a view function for the template view with caching enabled.
+        """
+        return cache_page(int(os.environ["CACHE_TTL"]))(
+            super(CachedTemplateView, cls).as_view(**initkwargs)
+        )
+
+
+class NeverCacheMixin(TemplateView):
+    """Mixin class to disable caching for Django views."""
+
+    @method_decorator(never_cache)
+    def dispatch(self, *args: Any, **kwargs: Any) -> HttpResponse:
```

Ruff (0.12.2)

Dynamically typed expressions (typing.Any) are disallowed in `*args`

(ANN401)